lib/storage/providers/SQLiteStorage.ts

```diff
@@ -85,84 +58,45 @@ const provider = {
              SET valueJSON = JSON_PATCH(valueJSON, JSON(:value));
         `;
 
-        const nonNullishPairs = _.filter(pairs, (pair) => !_.isUndefined(pair[1]));
-        const queryArguments = _.map(nonNullishPairs, (pair) => {
+        const nonNullishPairs = pairs.filter((pair) => pair[1] !== undefined);
+        const queryArguments = nonNullishPairs.map((pair) => {
             const value = JSON.stringify(pair[1]);
             return [pair[0], value];
         });
 
         return db.executeBatchAsync([[query, queryArguments]]);
     },
-
-    /**
-     * Merges an existing value with a new one by leveraging JSON_PATCH
-     * @param {String} key
-     * @param {*} changes - the delta for a specific key
-     * @return {Promise<void>}
-     */
     mergeItem(key, changes) {
-        return this.multiMerge([[key, changes]]);
+        return this.multiMerge([[key, changes]]) as Promise<BatchQueryResult>;
     },
-
-    /**
-     * Returns all keys available in storage
-     * @returns {Promise<String[]>}
-     */
     getAllKeys: () =>
         db.executeAsync('SELECT record_key FROM keyvaluepairs;').then(({rows}) => {
             // eslint-disable-next-line no-underscore-dangle
-            const result = _.map(rows._array, (row) => row.record_key);
-            return result;
+            const result = rows?._array.map((row) => row.record_key);
```

Comment:
What happens here if rows in null? Is it returning the same as before?

Reply:
You are right, before it was an empty array, after this change it resulted in undefined, thank you! Adjusted in last commit.